test/scenarios/CommunicationSimulators/sumo/tiergarten.rou.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<!-- Generated by VSimRTI -->
```

Comment:
remove "generated by VSimRTI"

tabs -> spaces in all xml-files (nod, edg, con, rou.xml)